docs/hsm/README.md

```diff
@@ -0,0 +1,313 @@
+---
+title: HSM Support
+weight: 690
+---
+
+Bank-Vaults offers a lot of alternatives for encrypting and storing the `unseal-keys` and the `root-token` for Vault. One of the encryption technics is the HSM - Hardware Security Module. HSM offers an industry-standard way to encrypt your data in on-premise environments.
+
+You can use a Hardware Security Module (HSM) to generate and store the private keys used by Bank-Vaults. Some articles still point out the speed of HSM devices as their main selling point, but an average PC can do more cryptographic operations. Rather the main benefit is from the security point of view. An HSM protects your private keys and handles cryptographic operations, which allows the encryption of protected information without exposing the private keys (they are not extractable). Bank-Vaults currently supports the [PKCS11](https://en.wikipedia.org/wiki/PKCS_11) software standard to communicate with an HSM. Fulfilling compliance requirements (for example, PCI DSS) is also a great benefit of HSMs, so from now on you can achieve that with Bank-Vaults.
+
+## Implementation in Bank-Vaults
+
+To support HSM devices for encrypting unseal-keys and root-tokens, Bank-Vaults:
+
+- implements an encryption/decryption `Service` named `hsm` in the `bank-vaults` CLI,
+- the `bank-vaults` Docker image now includes the SoftHSM (for testing) and the OpenSC tooling,
+- the operator is aware of HSM and its nature.
+
+The HSM offers an encryption mechanism, but the unseal-keys and root-token have to be stored somewhere after they got encrypted. Some HSM devices offer to store a limited quantity of arbitrary data (like Nitrokey HSM). Bank-Vaults additionally offers Kubernetes Secrets to storage backend the encrypted unseal keys. We think that after encryption is safe to store the Kubernetes Secrets in this form. In future versions, we will probably offer more storage backends, but currently, this fulfills our requirements.
```

Comment:
I don't get this sentence, do we mean that with Bank-Vaults, the user can store the encrypted unseal keys as Kubernetes Secrets? Like: 
With Bank-Vaults, you can also store the encrypted unseal keys as Kubernetes Secrets. We think that after encryption, it is safe to ...